core/server/data/migrations/versions/4.17/02-add-offers-table.js

```diff
@@ -0,0 +1,17 @@
+const utils = require('../../utils');
+
+module.exports = utils.addTable('offers', {
+    id: {type: 'string', maxlength: 24, nullable: false, primary: true},
+    name: {type: 'string', maxlength: 191, nullable: false},
+    slug: {type: 'string', maxlength: 191, nullable: false, unique: true},
```

Comment:
Is this to store the Code for an Offer?

Reply:
Yeah this is to store the offer code, i realized the offer code is essentially as good as an offer slug since it will have all of the slug properties for a URL, so we can just store it as that. Was split between naming it `slug` or something like `code`, but `slug` seemed more consistent to me so went with that, not a strong pref tho, wdyt ?

Reply:
It wasn't immediately obvious to me what this was storing - but happy to leave it like this if it makes sense to you

Reply:
It not being immediately obvious is definitely a problem, maybe it makes more sense to use the name as `code` in this case so we don't have same problem down the link. I'll rename it to `code` so its more clear, we can always make it behave like a slug behind the scenes :)

Reply:
Done